python/py-graph-tool/Portfile

```diff
@@ -25,11 +25,11 @@ homepage            https://graph-tool.skewed.de
 master_sites        https://downloads.skewed.de/graph-tool/
 use_bzip2           yes
 
-checksums           rmd160  786b809ef049c25e08ce549d96f19efdfd7c2e1f \
-                    sha256  5f1bee094220cfb24868db0faf47f8e2ffd26f77dd709995842e4d9e898da86c \
-                    size    15170272
+checksums           rmd160  316e8cc9c0b15e066e8ac2388a460ba4866505fe \
+                    sha256  305508f0c2989a150aecd5df010424979cd17e83f67852121e4ab29eb07d3275 \
+                    size    15189665
 
-boost.version       1.71
+boost.version       1.76
 
 python.versions     37 38 39 310
```

Comment:
is Python 3.11, currently MacPorts default version, supported? If so, please add that subport.

Reply:
py311 isn’t supported because its dependency `py311-gobject3` isn’t supported, and a trivial addition of `py311` to this dependency fails to install.